receiver/vcenterreceiver/processors.go

```diff
@@ -15,25 +15,59 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/vcenterreceiver/internal/metadata"
 )
 
+type DatacenterStats struct {
+	ClusterStatus  map[types.ManagedEntityStatus]int64
+	HostStats      map[string]map[types.ManagedEntityStatus]int64
```

Comment:
HostStatusCount

Reply:
Since the Host and VM are technically more than just a status, they also include power states, I was wondering if we would still want to name these variables as such?

Reply:
resolve